mappings/net/minecraft/server/config/BannedPlayerList.mapping

```diff
@@ -1,3 +1,3 @@
-CLASS xd net/minecraft/server/config/BannedProfilesList
+CLASS xd net/minecraft/server/config/BannedPlayerList
```

Comment:
Not sure about this, Any reason you didnt pick `BannedProfileList`?

Reply:
Yeah because everywhere else they are called banned players.